README.md

```diff
@@ -142,9 +142,9 @@ module.exports = {
 
 ## Importing documents from another locale
 
-Exported documents maintain their locale settings. If the locale during import differs from the export locale, and only one locale is configured in the `@apostrophecms/i18n` module, the documents will be automatically rewritten to align with the new import locale.
+Exported documents maintain their locale settings. If the locale during import differs from the export locale, and only one locale is configured in the `@apostrophecms/i18n` module, the document path will be automatically rewritten to align with the new import locale.
```

Comment:
I'd replace this whole paragraph with:

Regardless of the original locale, imported documents are imported into the current locale. You will be prompted to confirm if the locale is different and you do not have, or are not opting to use, the automatic translation module.

Reply:
Changed.